packages/slider/src/presenters/SliderPresenter.js

```diff
@@ -40,7 +67,28 @@ export default class SliderPresenter extends Component {
             resolvedRoles
           );
 
-          return (
+          const marks =
+            variant === sliderTypes.DISCRETE
+              ? this.getTickMarks(min, max, otherProps.step, styles)
+              : null;
+
+          return marks ? (
+            <div className={css(styles.discrete)}>
+              <input
+                className={css(styles.slider)}
+                disabled={disabled}
+                type="range"
+                aria-valuemin={min}
+                aria-valuemax={max}
+                aria-valuenow={value}
+                max={max}
+                min={min}
+                value={value}
+                {...otherProps}
+              />
+              <div className={css(styles.markContainer)}>{marks}</div>
```

Comment:
Instead of conditionally rendering the entire component, we can just do that for the `markContainer` with the same condition.

Reply:
I suggest to keep current render as for discrete slider, another parent DOM is added to input and markContainer which is not needed for original slider.

Reply:
That is a good point, I missed the wrapping <div> before, you can ignore this comment.